packages/uui-textarea/lib/uui-textarea.element.ts

```diff
@@ -333,6 +255,94 @@ export class UUITextareaElement extends FormControlMixin(LitElement) {
       </textarea>
     `;
   }
+
+  static styles = [
+    css`
+      :host {
+        position: relative;
+        --textarea-font-size: var(--uui-size-5);
```

Comment:
In uui-input we set the font-size to `font-size: inherit;`. Did you test if this was a possibility? Cause then I think we should aim to do the same on both for consistency. That would enable the font-size set outside to affect this, which I think is okay as that's what we do on uui-input. 

Also I think its best to not move the style part of this component, since we still have the style part at the top of the other components in UI Library. So for the sake of begin consistent I would like to it stay, and then we instead should make one PR that moves all of them. And sets the ES-Lint rule similar to in Backoffice, to enforce it.